pkg/test/resourcefixture/testdata/basic/workstations/workstationconfig/workstationconfig-full/_generated_object_workstationconfig-full.golden.yaml

```diff
@@ -1,6 +1,8 @@
-apiVersion: workstations.cnrm.cloud.google.com/v1alpha1
+apiVersion: workstations.cnrm.cloud.google.com/v1beta1
 kind: WorkstationConfig
 metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
```

Comment:
Do you know why this annotation is applied only in beta?

Reply:
I think this is a bug. The management conflict annotation defaulter webhook is incorrectly applying the annotation for direct resources. Fixed in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3417